packages/wonder-blocks-core/src/hooks/use-on-mount-effect.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import * as React from "react";
+
+/**
+ * Runs a callback once on mount.
+ *
+ * If the callback returns a cleanup function, it will be called with the component is unmounted.
```

Comment:
**suggestion:** I think it's worth including in this commentary a snippet of code showing the equivalent `useEffect` code.

Reply:
It felt redudant b/c the code is right below, but I can see people just reading the comment and not looking the code.  I'll add some comment about it.

Reply:
Folks using this in their code are going to see the function comment in the intellisense tooltip - they aren't going to see the code.